cartridges/int_adyen_overlay/cartridge/templates/default/threeds2/adyen3ds2.isml

```diff
@@ -0,0 +1,114 @@
+<isset name="resultCode" value="${pdict.resultCode}" scope="page"/>
+<isset name="token3ds2" value="${pdict.token3ds2}" scope="page"/>
+<isset name="authorise3DS2" value="${URLUtils.https('Adyen-Authorize3DS2')}" scope="page"/>
+
+<html>
+<head>
+    <script type="text/javascript" src="https://checkoutshopper-test.adyen.com/checkoutshopper/sdk/2.5.0/adyen.js"></script>
```

Comment:
Can you use AdyenHelper.getLoadingContext() to get the correct context: TEST/LIVE?

Reply:
Yes, good one.